starter-core/src/main/java/io/micronaut/starter/feature/function/template/http/httpFunctionJavaController.rocker.raw

```diff
@@ -0,0 +1,49 @@
+@import io.micronaut.starter.application.Project
+
+@args (
+Project project
+)
+
+@if (project.getPackageName() != null) {
+package @project.getPackageName();
+}
+
+
+import io.micronaut.http.annotation.*;
+
+@@Controller("/")
+public class @project.getClassName()Controller {
+
+    @@Get(uri="/", produces="text/plain")
```

Comment:
We should use the constants defined in `MediaType` instead of `text/plain` here.

Reply:
Instead of: 

```java
@Get(uri="/", produces=["text/plain"])
```

I have rewritten to: 

```java
import io.micronaut.http.annotation.Produces;
import io.micronaut.http.MediaType;

...

    @Produces(MediaType.TEXT_PLAIN)
    @Get
```